hare/flows_test.go

```diff
@@ -401,13 +401,19 @@ func Test_multipleCPs(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	mesh, err := mocknet.FullMeshLinked(totalNodes)
+	defer func() {
+		err := mesh.Close()
+		require.NoError(t, err)
+	}()
```

Comment:
This should be after checking `err` to be `nil`, or it might cause unexpected behavior.

Reply:
Thank you :+1: